README.md

```diff
@@ -27,10 +27,20 @@ cordova plugin add https://github.com/samzilverberg/cordova-mixpanel-plugin.git
 **window.mixpanel.people:**
 
 - identify(distinctId, onSuccess, onFail)
-- registerPushToken(apnsOrGcmToken, onSuccess, onFail)
+- registerPushId(registrationId, onSuccess, onFail)
```

Comment:
i got some time to work on this now, will merge and do the rename stuff directly on master